react-rendering-lists/src/index.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+function PokemonList(props) {
+  const pokedex = props.pokedex;
+  const listPokedex = pokedex.map(pokedex =>
+    <li key={pokedex.name}>{pokedex.name}</li>
```

Comment:
We usually want to use an `id` for the `key` property, in this case the `number` property in your pokedex items.